### ISSUE TITLE
Is it possible to use RNN before CNN

### ISSUE THREAD

**Issue by lqj1990** [fchollet/keras#2198]
If I want to use RNN before CNN, what can I do? 
For example, I have some sequence of different length :[x11,x12,x13,...x1N],[x21,x22,x23,.....x2M]....
so I want to use RNN to get the output of each sequence y = [y1,y2,....], and then use CNN to label y. 、

**Comment by jrieke** [comment]
Use the parameter `return_sequences=False` in your last RNN layer ([docs](http://keras.io/layers/recurrent/)). The RNN will then yield one output for the complete sequence, which you can feed into your CNN.

**Comment by philipperemy** [comment]
Yes it's possible. I think you need

```
TimeDistributedMerge
```
from http://keras.io/layers/core/

Input shape
3D tensor with shape: (samples, steps, features).

Output shape
2D tensor with shape: (samples, features).

It becomes easy to stack CNN layers on top of that.

**Comment by lqj1990** [comment]
I think TImeDistributeDense is used to deal with every step output of RNN. It's different from my case.
In my case, I have a sequence [x11,x12,x13]. After processed by RNN, it will output a vector y1. Another sequence [x21, x22, x23, x24] will get a vector y2 after processed by RNN. Then y1 and y2 will be used as input of a CNN to get a label L1. So I don't know how to do it.

**Comment by philipperemy** [comment]
Why don't you use return_sequences=True inside your RNN layers? It will return a sequence that you can input to your CNN.

**Comment by lqj1990** [comment]
It will get every step of RNN， but I don't need to do that. In fact I want to use R(recursive)NN, but I don't know how to realize it, so I decide to try RNN.

**Comment by philipperemy** [comment]
You can have a look at this long conversation on how to use TimeDistributedDense:

https://github.com/fchollet/keras/issues/1029
